chalice_app/chalicelib/static/index.html

```diff
@@ -2679,6 +2820,20 @@ <h1>Predict with SageMaker Model</h1>
         Predict Price (AUD$/wk)
       </button>
       <div id="message" class="result"></div>
+      <div class="container" id="documentation">
+    <h2>Documentation</h2>
+    <div class="documentation-section">
+      <h3>Project Documentation</h3>
+      <p>For detailed project documentation, please visit <a href="https://docs.google.com/document/d/1j7SKMtpVIPzhOkzZ7TffQCHxCKNeilz7gkUAR5KA22g/edit#heading=h.xdi7upsffmgs" target="_blank">this link</a>.</p>
```

Comment:
Make sure that only our team can edit this file. But also make it so that anyone with the link can view it